Move to lowest un-prescaled triggers

dntaylor opened the pull request:
Instead of Ele23, moving to Ele17. I thought it was this the whole time... guess not.
@nwoods Make sure to check this, if ZZ really wants the Ele23 trigger, you will need to override it in the zz.py.

dntaylor committed:
updates for metnohf

dntaylor committed:
modify to work with 74X X>=12

dntaylor committed:
latest json

dntaylor committed:
and v2

dntaylor committed:
fix to trigger

dntaylor committed:
fix hzz id merge

dntaylor committed:
fix to the recommended lepton separation

Commit:
Keep HZZ ID and isolation decisions (no FSR) in ntuples.

nwoods commented:
This looks OK to me. Would you mind cherry-picking dc09d25bc5c7720ffd127e08692d2a1aa262b459 for me? It's too small a change for its own PR but it would be nice to have.

dntaylor committed:
reminiaod of run2015D, still need run2015C

dntaylor committed:
add met shifts in miniaod

dntaylor committed:
add scale down too

dntaylor committed:
add n-1 functionality to vid

dntaylor committed:
latest

dntaylor commented on NtupleTools/python/embedElectronIDs.py:
```diff
@@ -47,6 +47,26 @@ def embedElectronIDs(process, use25ns, eSrc):
         ]
 
+    # N-1 results
+    nMinusOneNames = ['GsfEleEffAreaPFIsoCut_0']
```
@lgray Why is this the name of the cut? When I look here: https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_tools.py#L262 I see without the "_0".

dntaylor committed:
bug fix

dntaylor committed:
add MT and dPhi

dntaylor committed:
consistent naming

dntaylor committed:
golden json 848.7/pb

dntaylor committed:
updated met

dntaylor committed:
adding met filters

dntaylor commented:
I've add MET filters and the latest golden json. Also added support for n-1 VID studies.

Some things to note:

If you use MET, I've renamed some things.

dntaylor committed:
adding bools for trig isolation
